programs/server/config.xml

```diff
@@ -1444,7 +1451,10 @@
          obtain an encryption key at startup if such a command is
          defined, or encryption codecs will be disabled otherwise. The
          command is executed through /bin/sh and is expected to write
-         a Base64-encoded key to the stdout. -->
+         a Base64-encoded key to the stdout.
+
+         Note: ClickHouse Cloud https://clickhouse.cloud/ allows encryption with customer-managed keys.
```

Comment:
```suggestion
         Note: ClickHouse Cloud https://clickhouse.cloud/ supports encryption with customer-managed keys.
```
I feel this might be better.